Simplenote/Classes/UIColorName.swift

```diff
@@ -100,6 +101,8 @@ extension UIColorName {
             return .searchHighlightFontColor
         case .simplenoteAlmostBlack:
             return .simplenoteAlmostBlack
+        case .simplenoteGunmetal:
```

Comment:
Out of scope for this PR, but I am wondering if we should eventually adopt the color studio names for the colors we use...

Reply:
Studio Names == Zeplin right?

I've **tried** to adopt names similar to iOS Dynamic Fonts. `Color for Header` etc etc etc. BUT those new names come from the Zeplin mockups. I'd go with whatever is easier to maintain, definitely!!